### PR TITLE
Add required BGS Rating endpoints

### PR DESCRIPTION
**Pull request by shanear**
connects https://github.com/department-of-veterans-affairs/caseflow/issues/4874

example response from UAT for `rating.find_by_participant_id_and_date_range()`:

```
{:error_level=>"0",
 :any_ratings_within_range_flag=>"Y",
 :combined_percent_current_list=>
  {:combined_percent_current=>
    [{:beneficiary_participant_id=>"600185103",
      :combined_pct_without_reca=>"70",
      :combined_pct_without_severance=>"70",
      :combined_pct_without_tort=>"70",
      :combined_pct_without_workers_comp=>"70",
      :compensation_service_period_type=>"7",
      :effective_date=>
       #<DateTime: 2013-06-01T00:00:00-05:00 ((2456445j,18000s,0n),-18000s,2299161j)>,
      :indiv_unemployability_ind=>"Y",
      :not_service_conn_bilateral_value=>"0.0",
      :not_service_conn_combined_pct=>"70",
      :reca_withholding_pct=>"0",
      :service_conn_bilateral_percent=>"0.0",
      :service_conn_combined_pct=>"70",
      :severance_withholding_pct=>"0",
      :tort_withholding_pct=>"0",
      :veteran_participant_id=>"600185103",
      :workers_comp_withholding_pct=>"0"},
     {:beneficiary_participant_id=>"600185103",
      :combined_pct_without_reca=>"70",
      :combined_pct_without_severance=>"70",
      :combined_pct_without_tort=>"70",
      :combined_pct_without_workers_comp=>"70",
      :compensation_service_period_type=>"7",
      :effective_date=>
       #<DateTime: 2018-03-01T00:00:00-06:00 ((2458179j,21600s,0n),-21600s,2299161j)>,
      :indiv_unemployability_ind=>"Y",
      :not_service_conn_bilateral_value=>"0.0",
      :not_service_conn_combined_pct=>"70",
      :reca_withholding_pct=>"0",
      :service_conn_bilateral_percent=>"0.0",
      :service_conn_combined_pct=>"70",
      :severance_withholding_pct=>"0",
      :tort_withholding_pct=>"0",
      :veteran_participant_id=>"600185103",
      :workers_comp_withholding_pct=>"0"}]},
 :differences_found=>"N",
 :earliest_dependency_eligibility_date=>
  #<DateTime: 2013-06-01T00:00:00-05:00 ((2456445j,18000s,0n),-18000s,2299161j)>,
 :multiple_ratings_flag=>"Y",
 :prfil_dt=>
  #<DateTime: 2018-04-04T14:20:05-05:00 ((2458213j,69605s,0n),-18000s,2299161j)>,
 :promulgation_dt=>
  #<DateTime: 2018-04-04T14:24:10-05:00 ((2458213j,69850s,0n),-18000s,2299161j)>,
 :ptcpnt_vet_id=>"600185103",
 :rating_details=>
  [{:error_level=>"0",
    :effective_dt=>
     #<DateTime: 2013-06-01T00:00:00-05:00 ((2456445j,18000s,0n),-18000s,2299161j)>,
    :sc_percent=>"70"},
   {:error_level=>"0",
    :effective_dt=>
     #<DateTime: 2018-03-01T00:00:00-06:00 ((2458179j,21600s,0n),-21600s,2299161j)>,
    :sc_percent=>"70"}],
 :rating_profile_list=>
  {:rating_profile=>
    [{:cdsht_dcmnt_id=>"684046",
      :client_rating_dt=>
       #<DateTime: 2017-10-10T12:53:36-05:00 ((2458037j,64416s,0n),-18000s,2299161j)>,
      :comp_id=>
       {:prfil_dt=>
         #<DateTime: 2017-10-10T12:47:48-05:00 ((2458037j,64068s,0n),-18000s,2299161j)>,
        :ptcpnt_vet_id=>"600185103"},
      :dcmnt_title_txt=>"Rating Decision",
      :jrn_dt=>
       #<DateTime: 2017-10-24T15:12:22-05:00 ((2458051j,72742s,0n),-18000s,2299161j)>,
      :jrn_lctn_id=>"328",
      :jrn_obj_id=>"VBMS  - PACL",
      :jrn_status_type_cd=>"U",
      :jrn_user_id=>"HE328_ASUPER",
      :jrsdtn_txt=>"Original Disability Claim Received",
      :locked_ind=>"N",
      :narrtv_dcmnt_id=>"684047",
      :prmlgn_dt=>
       #<DateTime: 2017-10-24T15:12:22-05:00 ((2458051j,72742s,0n),-18000s,2299161j)>,
      :prmlng_claim_id=>"600113228",
      :rating_dt=>
       #<DateTime: 2017-10-10T12:53:37-05:00 ((2458037j,64417s,0n),-18000s,2299161j)>,
      :rating_web_ind=>"Y",
      :rba_user_id=>"VBACOWHYNOC",
      :rcvd_dt=>
       #<DateTime: 2013-05-22T00:00:00-05:00 ((2456435j,18000s,0n),-18000s,2299161j)>,
      :system_type_cd=>"C"},
     {:cdsht_dcmnt_id=>"700494",
      :client_rating_dt=>
       #<DateTime: 2018-04-04T14:22:17-05:00 ((2458213j,69737s,0n),-18000s,2299161j)>,
      :comp_id=>
       {:prfil_dt=>
         #<DateTime: 2018-04-04T14:20:05-05:00 ((2458213j,69605s,0n),-18000s,2299161j)>,
        :ptcpnt_vet_id=>"600185103"},
      :dcmnt_title_txt=>"Rating Decision",
      :jrn_dt=>
       #<DateTime: 2018-04-04T14:24:10-05:00 ((2458213j,69850s,0n),-18000s,2299161j)>,
      :jrn_lctn_id=>"328",
      :jrn_obj_id=>"VBMS  - PACL",
      :jrn_status_type_cd=>"U",
      :jrn_user_id=>"T2328_AUTH",
      :jrsdtn_txt=>"New Claim Received",
      :locked_ind=>"N",
      :narrtv_dcmnt_id=>"700495",
      :prmlgn_dt=>
       #<DateTime: 2018-04-04T14:24:10-05:00 ((2458213j,69850s,0n),-18000s,2299161j)>,
      :prmlng_claim_id=>"600127329",
      :rating_dt=>
       #<DateTime: 2018-04-04T14:22:15-05:00 ((2458213j,69735s,0n),-18000s,2299161j)>,
      :rating_web_ind=>"Y",
      :rba_user_id=>"T1317_RVSR",
      :rcvd_dt=>
       #<DateTime: 2018-01-01T00:00:00-06:00 ((2458120j,21600s,0n),-21600s,2299161j)>,
      :system_type_cd=>"C"}]}}
```

example response from `rating_profile.find`

```
{:call_id=>"17",
 :jrn_dt=>
  #<DateTime: 2017-10-24T15:12:22-05:00 ((2458051j,72742s,0n),-18000s,2299161j)>,
 :jrn_lctn_id=>"328",
 :jrn_obj_id=>"VBMS  - PACL",
 :jrn_stt_tc=>"U",
 :jrn_user_id=>"HE328_ASUPER",
 :row_cnt=>"1",
 :row_id=>"8",
 :ancillary_decisions=>
  {:call_id=>"17",
   :jrn_dt=>
    #<DateTime: 2017-10-10T12:53:44-05:00 ((2458037j,64424s,0n),-18000s,2299161j)>,
   :jrn_lctn_id=>"317",
   :jrn_obj_id=>"cp_ancillary2_pkg.do_create",
   :jrn_stt_tc=>"I",
   :jrn_user_id=>"VBACOWHYNOC",
   :parent_id=>"8",
   :parent_name=>"CP_RBA_PRFIL2",
   :row_cnt=>"1",
   :row_id=>"1",
   :anc_decn_tc=>"PT35",
   :anc_decn_tn=>"Basic Eligibility under 38 USC Ch 35",
   :begin_dt=>
    #<DateTime: 2013-05-22T00:00:00-05:00 ((2456435j,18000s,0n),-18000s,2299161j)>,
   :prfl_dt=>
    #<DateTime: 2017-10-10T12:47:48-05:00 ((2458037j,64068s,0n),-18000s,2299161j)>,
   :ptcpnt_decn_id=>"600185103",
   :ptcpnt_vet_id=>"600185103",
   :rating_dt=>
    #<DateTime: 2017-10-10T12:53:37-05:00 ((2458037j,64417s,0n),-18000s,2299161j)>,
   :rating_sn=>"113897",
   :rba_issue_id=>"101186"},
 :associated_claims=>
  {:call_id=>"17",
   :jrn_dt=>
    #<DateTime: 2017-10-10T12:47:51-05:00 ((2458037j,64071s,0n),-18000s,2299161j)>,
   :jrn_lctn_id=>"317",
   :jrn_obj_id=>"vn_rba_clm_pkg.do_create",
   :jrn_stt_tc=>"I",
   :jrn_user_id=>"VBACOWHYNOC",
   :parent_id=>"8",
   :parent_name=>"CP_RBA_PRFIL2",
   :row_cnt=>"1",
   :row_id=>"8",
   :bnft_clm_tc=>"110LCOMP",
   :bnft_clm_tn=>"Initial Live Comp/Pension",
   :claim_rcvd_dt=>
    #<DateTime: 2013-05-22T00:00:00-05:00 ((2456435j,18000s,0n),-18000s,2299161j)>,
   :clm_id=>"600113228",
   :clmnt_nm=>"FIVE NELSON",
   :end_prdct_type_cd=>"110",
   :locked_ind=>"N",
   :prfl_dt=>
    #<DateTime: 2017-10-10T12:47:48-05:00 ((2458037j,64068s,0n),-18000s,2299161j)>,
   :ptcpnt_clmnt_id=>"600185103",
   :ptcpnt_vet_id=>"600185103",
   :rating_dt=>
    #<DateTime: 2017-10-10T12:53:37-05:00 ((2458037j,64417s,0n),-18000s,2299161j)>,
   :rba_clm_sn=>"80856",
   :rba_user_nm=>"HELEN SHTUKA, 328",
   :status_tc=>"CLR"},
 :cdsht_dcmnt_id=>"684046",
 :client_rating_dt=>
  #<DateTime: 2017-10-10T12:53:36-05:00 ((2458037j,64416s,0n),-18000s,2299161j)>,
 :dcmnt_title_txt=>"Rating Decision",
 :disabilities=>
  {:call_id=>"17",
   :jrn_dt=>
    #<DateTime: 2018-04-04T14:22:27-05:00 ((2458213j,69747s,0n),-18000s,2299161j)>,
   :jrn_lctn_id=>"317",
   :jrn_obj_id=>"VBMS-cp_dsblty2_pkg.do_update",
   :jrn_stt_tc=>"U",
   :jrn_user_id=>"T1317_RVSR",
   :parent_name=>"CP_RBA_PRFIL2",
   :row_cnt=>"1",
   :row_id=>"1",
   :additional_decision_bases=>
    {:call_id=>"17",
     :jrn_dt=>
      #<DateTime: 2017-10-10T12:53:44-05:00 ((2458037j,64424s,0n),-18000s,2299161j)>,
     :jrn_lctn_id=>"317",
     :jrn_obj_id=>"VBMS-cp_dsblty_decn_pkg.do_creat",
     :jrn_stt_tc=>"I",
     :jrn_user_id=>"VBACOWHYNOC",
     :parent_id=>"1",
     :parent_name=>"CP_DSBLTY2",
     :row_cnt=>"1",
     :row_id=>"4",
     :dsblty_decn_basis_tc=>"INC",
     :dsblty_decn_sn=>"15410",
     :dsblty_dt=>
      #<DateTime: 2017-10-10T12:53:44-05:00 ((2458037j,64424s,0n),-18000s,2299161j)>,
     :dsblty_id=>"1114008"},
   :combat_ind=>"N",
   :decn_basis_tc=>"INC",
   :decn_basis_tn=>"Incurred",
   :decn_tc=>"SVCCONNCTED",
   :decn_tn=>"Service Connected",
   :dis_dt=>
    #<DateTime: 2017-10-10T12:53:44-05:00 ((2458037j,64424s,0n),-18000s,2299161j)>,
   :dis_sn=>"1114008",
   :disability_evaluations=>
    {:call_id=>"17",
     :jrn_dt=>
      #<DateTime: 2018-04-04T14:22:27-05:00 ((2458213j,69747s,0n),-18000s,2299161j)>,
     :jrn_lctn_id=>"317",
     :jrn_obj_id=>"cp_dsblty_eval2_pkg.do_update",
     :jrn_stt_tc=>"U",
     :jrn_user_id=>"VBACOWHYNOC",
     :parent_id=>"1",
     :parent_name=>"CP_DSBLTY2",
     :row_cnt=>"1",
     :row_id=>"3",
     :begin_dt=>
      #<DateTime: 2013-05-22T00:00:00-05:00 ((2456435j,18000s,0n),-18000s,2299161j)>,
     :conv_begin_dt=>
      #<DateTime: 2013-06-01T00:00:00-05:00 ((2456445j,18000s,0n),-18000s,2299161j)>,
     :dgnstc_tc=>"9411",
     :dgnstc_tn=>"Post traumatic stress disorder",
     :dgnstc_txt=>"PTSD",
     :dis_dt=>
      #<DateTime: 2017-10-10T12:53:44-05:00 ((2458037j,64424s,0n),-18000s,2299161j)>,
     :dis_sn=>"1114008",
     :efctv_dt_ovride_tc=>"DEFAULT",
     :efctv_dt_ovride_tn=>"Default",
     :inactv_dt=>
      #<DateTime: 2018-04-04T14:20:05-05:00 ((2458213j,69605s,0n),-18000s,2299161j)>,
     :major_ind=>"N",
     :ovride_action_ind=>"N",
     :prcnt_no=>"70",
     :prev_evaltn_ind=>"N",
     :prfl_dt=>
      #<DateTime: 2017-10-10T12:47:48-05:00 ((2458037j,64068s,0n),-18000s,2299161j)>,
     :ptcpnt_id_a=>"600185103",
     :rating_dt=>
      #<DateTime: 2017-10-10T12:53:37-05:00 ((2458037j,64417s,0n),-18000s,2299161j)>,
     :rating_sn=>"113899",
     :rba_issue_id=>"101185",
     :schedule_tc=>"CS",
     :schedule_tn=>"Current Schedule"},
   :exam_at_once_ind=>"N",
   :inactv_dt=>
    #<DateTime: 2018-04-04T14:20:05-05:00 ((2458213j,69605s,0n),-18000s,2299161j)>,
   :indvdl_unempl_ind=>"N",
   :prfl_dt=>
    #<DateTime: 2017-10-10T12:47:48-05:00 ((2458037j,64068s,0n),-18000s,2299161j)>,
   :ptcpnt_id_a=>"600185103",
   :static_ind=>"Y",
   :svc_per_tc=>"7",
   :svc_per_tn=>"Vietnam Era"},
 :ind_unemployment_decisions=>
  {:call_id=>"17",
   :jrn_dt=>
    #<DateTime: 2017-10-10T12:53:44-05:00 ((2458037j,64424s,0n),-18000s,2299161j)>,
   :jrn_lctn_id=>"317",
   :jrn_obj_id=>"cp_indv_unempl2_pkg.do_create",
   :jrn_stt_tc=>"I",
   :jrn_user_id=>"VBACOWHYNOC",
   :parent_id=>"8",
   :parent_name=>"CP_RBA_PRFIL2",
   :row_cnt=>"1",
   :row_id=>"0",
   :begin_dt=>
    #<DateTime: 2013-05-22T00:00:00-05:00 ((2456435j,18000s,0n),-18000s,2299161j)>,
   :conv_begin_dt=>
    #<DateTime: 2013-06-01T00:00:00-05:00 ((2456445j,18000s,0n),-18000s,2299161j)>,
   :efctv_dt_ovride_tc=>"DEFAULT",
   :efctv_dt_ovride_tn=>"Default",
   :iu_decn_tc=>"18A",
   :iu_decn_tn=>"Individual Unemployability Granted",
   :ovride_action_ind=>"N",
   :prfl_dt=>
    #<DateTime: 2017-10-10T12:47:48-05:00 ((2458037j,64068s,0n),-18000s,2299161j)>,
   :ptcpnt_id_a=>"600185103",
   :rating_dt=>
    #<DateTime: 2017-10-10T12:53:37-05:00 ((2458037j,64417s,0n),-18000s,2299161j)>,
   :rating_sn=>"113898",
   :rba_issue_id=>"101184"},
 :jrsdtn_txt=>"Original Disability Claim Received",
 :locked_ind=>"N",
 :narrtv_dcmnt_id=>"684047",
 :prfl_dt=>
  #<DateTime: 2017-10-10T12:47:48-05:00 ((2458037j,64068s,0n),-18000s,2299161j)>,
 :prmlgn_dt=>
  #<DateTime: 2017-10-24T15:12:22-05:00 ((2458051j,72742s,0n),-18000s,2299161j)>,
 :prmlng_claim_id=>"600113228",
 :ptcpnt_vet_id=>"600185103",
 :rating_award_details=>
  {:call_id=>"17",
   :jrn_dt=>
    #<DateTime: 2017-10-10T12:53:45-05:00 ((2458037j,64425s,0n),-18000s,2299161j)>,
   :jrn_lctn_id=>"317",
   :jrn_obj_id=>"cp_ratesumsrc_pkg.do_create",
   :jrn_stt_tc=>"I",
   :jrn_user_id=>"VBACOWHYNOC",
   :parent_id=>"8",
   :parent_name=>"CP_RBA_PRFIL2",
   :row_cnt=>"1",
   :row_id=>"9",
   :award_event_ind=>nil,
   :comp_svc_per_tc=>"7",
   :efctv_dt=>
    #<DateTime: 2013-06-01T00:00:00-05:00 ((2456445j,18000s,0n),-18000s,2299161j)>,
   :efctv_dt_ovride_ind=>"N",
   :inactv_dt=>
    #<DateTime: 2018-04-04T14:22:28-05:00 ((2458213j,69748s,0n),-18000s,2299161j)>,
   :iu_ind=>"Y",
   :nsc_percent=>"70",
   :per_wo_reca=>"70",
   :per_wo_sev=>"70",
   :per_wo_tort=>"70",
   :per_wo_wrk=>"70",
   :prfil_dt=>
    #<DateTime: 2017-10-10T12:47:48-05:00 ((2458037j,64068s,0n),-18000s,2299161j)>,
   :ptcpnt_id_a=>"600185103",
   :ptcpnt_vet_id=>"600185103",
   :rating_award_sumry_id=>"121860",
   :rating_dt=>
    #<DateTime: 2017-10-10T12:53:37-05:00 ((2458037j,64417s,0n),-18000s,2299161j)>,
   :rba_user_id=>"VBACOWHYNOC",
   :sc_bilat_no=>"0.0",
   :sc_percent=>"70",
   :smc_wthld_code=>nil},
 :rating_dt=>
  #<DateTime: 2017-10-10T12:53:37-05:00 ((2458037j,64417s,0n),-18000s,2299161j)>,
 :rating_issues=>
  [{:call_id=>"17",
    :jrn_dt=>
     #<DateTime: 2017-10-10T12:53:44-05:00 ((2458037j,64424s,0n),-18000s,2299161j)>,
    :jrn_lctn_id=>"317",
    :jrn_obj_id=>"cp_rba_issue2_pkg.do_create",
    :jrn_stt_tc=>"I",
    :jrn_user_id=>"VBACOWHYNOC",
    :parent_id=>"8",
    :parent_name=>"CP_RBA_PRFIL2",
    :row_cnt=>"3",
    :row_id=>"5",
    :anlys_gen_txt=>"RW50aXRsZW1lbnQgdG8gMu",
    :anlys_gen_txt_id=>"140789",
    :anlys_usr_txt=> "RW5XMu",
    :anlys_usr_txt_id=>"140788",
    :best_news_wt=>"2902",
    :copy_facts=>nil,
    :decn_row_id=>"101184",
    :decn_txt=>"Entitlement to individual unemployability is granted effective May 22, 2013.",
    :fact_usr_txt=>"gICAgICAgIC=",
    :issue_sort_seq_nbr=>"2",
    :isu_cat_tc=>"4",
    :isu_data_rec_id=>"101184",
    :isu_descr=>"Other - Individual Unemployability",
    :isu_tc=>"IU",
    :isu_txt=>"Entitlement to individual unemployability.",
    :prev_prfil_ind=>nil,
    :prfl_dt=>
     #<DateTime: 2017-10-10T12:47:48-05:00 ((2458037j,64068s,0n),-18000s,2299161j)>,
    :ptcpnt_id_a=>"600185103",
    :rba_anlys_id=>"104",
    :rba_issue_id=>"101184",
    :subjct_txt=>"Individual Unemployability",
    :temp_fact_txt=>"ICAgICAgICAgICA=",
    :vw_nm=>"VBACOWHYNOC"},
   {:call_id=>"17",
    :jrn_dt=>
     #<DateTime: 2017-10-10T12:53:44-05:00 ((2458037j,64424s,0n),-18000s,2299161j)>,
    :jrn_lctn_id=>"317",
    :jrn_obj_id=>"cp_rba_issue2_pkg.do_create",
    :jrn_stt_tc=>"I",
    :jrn_user_id=>"VBACOWHYNOC",
    :parent_id=>"8",
    :parent_name=>"CP_RBA_PRFIL2",
    :row_id=>"6",
    :anlys_gen_txt=>"U2VydmljZSBjb3duIG5hbWUu",
    :anlys_gen_txt_id=>"140791",
    :anlys_usr_txt=>"U2VydmljZUu",
    :anlys_usr_txt_id=>"140790",
    :best_news_wt=>"3010",
    :copy_facts=>nil,
    :decn_row_id=>"101185",
    :decn_txt=>"Service connection for PTSD is granted with an evaluation of 70 percent effective May 22, 2013.",
    :dis_sn=>"1114008",
    :fact_usr_txt=>"ICAgICAgICAg=",
    :issue_sort_seq_nbr=>"1",
    :isu_cat_tc=>"0",
    :isu_data_rec_id=>"101185",
    :isu_descr=>"Compensation - PTSD",
    :isu_txt=>"Service connection for PTSD.",
    :prev_prfil_ind=>nil,
    :prfl_dt=>
     #<DateTime: 2017-10-10T12:47:48-05:00 ((2458037j,64068s,0n),-18000s,2299161j)>,
    :ptcpnt_id_a=>"600185103",
    :rba_anlys_id=>"67",
    :rba_issue_id=>"101185",
    :subjct_txt=>"PTSD",
    :temp_fact_txt=>"ICAgICAg=",
    :vw_nm=>"VBACOWHYNOC"},
   {:call_id=>"17",
    :jrn_dt=>
     #<DateTime: 2017-10-10T12:53:44-05:00 ((2458037j,64424s,0n),-18000s,2299161j)>,
    :jrn_lctn_id=>"317",
    :jrn_obj_id=>"cp_rba_issue2_pkg.do_create",
    :jrn_stt_tc=>"I",
    :jrn_user_id=>"VBACOWHYNOC",
    :parent_id=>"8",
    :parent_name=>"CP_RBA_PRFIL2",
    :row_id=>"7",
    :anlys_gen_txt=>"RWxpZ2liaWx=",
    :anlys_gen_txt_id=>"140793",
    :anlys_usr_txt=>"RWxpZ2lia=",
    :anlys_usr_txt_id=>"140792",
    :best_news_wt=>"2732",
    :copy_facts=>nil,
    :decn_row_id=>"101186",
    :decn_txt=>
     "Basic eligibility to Dependents' Educational Assistance is established from May 22, 2013.",
    :fact_usr_txt=> "ICAgICAgICAgICAgICAgICAgIC=",
    :issue_sort_seq_nbr=>"3",
    :isu_cat_tc=>"4",
    :isu_data_rec_id=>"101186",
    :isu_descr=>"Other - Ancillary Decisions",
    :isu_tc=>"ANCDECN",
    :isu_txt=>
     "Eligibility to Dependents' Educational Assistance under 38 U.S.C. Chapter 35.",
    :prev_prfil_ind=>nil,
    :prfl_dt=>
     #<DateTime: 2017-10-10T12:47:48-05:00 ((2458037j,64068s,0n),-18000s,2299161j)>,
    :ptcpnt_id_a=>"600185103",
    :rba_anlys_id=>"78",
    :rba_issue_id=>"101186",
    :subjct_txt=>"Ancillary Decisions",
    :temp_fact_txt=>"ICAgI=",
    :vw_nm=>"VBACOWHYNOC"}],
 :rating_web_ind=>"Y",
 :rba_user_id=>"VBACOWHYNOC",
 :rcvd_dt=>
  #<DateTime: 2013-05-22T00:00:00-05:00 ((2456435j,18000s,0n),-18000s,2299161j)>,
 :system_tc=>"C"}
```

### CODE REVIEW

File: .rubocop.yml [hunk 1]
```diff
@@ -13,7 +13,7 @@ Naming:
   Enabled: false
 
 Metrics/LineLength:
-  Max: 100
+  Max: 120
```
Comment (shanear) [inline .rubocop.yml]: update this to match Caseflow's config

File: lib/bgs/base.rb [hunk 2]
```diff
@@ -135,7 +135,8 @@ def client
         ssl_cert_file: @ssl_cert_file,
         ssl_ca_cert_file: @ssl_ca_cert,
         open_timeout: 600, # in seconds
-        read_timeout: 600 # in seconds
+        read_timeout: 600, # in seconds
+        convert_request_keys_to: :none
```
Comment (shanear) [inline lib/bgs/base.rb]: Validated this doesn't affect any of our requests. But we should still do some UAT testing to validate.

File: lib/bgs/services.rb [hunk 3]
```diff
@@ -13,14 +13,16 @@
...
 require "bgs/services/benefit"
-require "bgs/services/veteran"
+require "bgs/services/claimant"
+require "bgs/services/org"
+require "bgs/services/person"
+require "bgs/services/rating"
+require "bgs/services/rating_profile"
 require "bgs/services/standard_data"
-require "bgs/services/address"
+require "bgs/services/veteran"
```
Comment (shanear) [inline lib/bgs/services.rb]: Alphabetical order woohoo 👐

Comment (sansb) [reply lib/bgs/services.rb]: bless

**Pull request merged**